crates/fj-host/src/model.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::result_large_err)]
```

Comment:
I think this is either to broad or too specific. We could decide that this warning is unreasonable. If that were the case, we should allow if for the whole crate. If we think the warning is reasonable, in general, then we should be more specific about disabling it. As it stands, it's unclear why this `#[allow(...)]` is here, and what specifically is affects.

Reply:
I'm not sure what to do about this one..  The warning ultimately stems from `fj_kernel::validate::ValidationError` being large (via `fj_host::model::Error::ShapeProcessor` that I added).

There are two identical comments about it in `fj-kernel/src/lib.rs` and `fj-operations/src/lib.rs`:
```
// I've made a simple change that put `ValidationError` over the threshold for
// this warning. I couldn't come up with an easy fix, and figured that silencing
// the warning is the most practical solution for now, as the validation
// infrastructure is in flux anyway. Maybe the problem will take care of itself.
#![allow(clippy::result_large_err)]
```

Maybe we should follow suit and copy that same comment and `#![allow(...)]` over to the `fj-host` and `fj-window` crates?

---

Another point of confusion is that I had put `#[allow(clippy::large_enum_variant)]` on `fj_host::model::Error`, but I don't get a warning when I remove it.  I don't remember now if I ever got a warning or if I was just trying something out in response to a rust-analyzer diagnostic message.  However, there is also `#[allow(clippy::large_enum_variant)]` on `fj_operations::shape_process::Error` and `fj_kernel::validate::ValidationError`, and I can remove both of those without generating a clippy warning.  Anyway, this paragraph might just be something to address as a separate issue.

Reply:
> There are two identical comments about it in `fj-kernel/src/lib.rs` and `fj-operations/src/lib.rs`:
> 
> ...
> 
> Maybe we should follow suit and copy that same comment and `#![allow(...)]` over to the `fj-host` and `fj-window` crates?

Sounds good to me. I don't know if that is the best solution, but it certainly is the good-enough solution to move this forward.


> Another point of confusion is that I had put `#[allow(clippy::large_enum_variant)]` on `fj_host::model::Error`, but I don't get a warning when I remove it. I don't remember now if I ever got a warning or if I was just trying something out in response to a rust-analyzer diagnostic message. However, there is also `#[allow(clippy::large_enum_variant)]` on `fj_operations::shape_process::Error` and `fj_kernel::validate::ValidationError`, and I can remove both of those without generating a clippy warning.

Maybe the recent Rust 1.66 release changed the behavior? It's also possible that `ValidationError` changed in the meantime and no longer triggers this warning. Not sure though. I've worked in that general area recently, but don't remember any specific changes to `ValidationError`.


> Anyway, this paragraph might just be something to address as a separate issue.

Sounds good. As I said in my review earlier, I'd have been happy to merge as initially submitted, and this is probably extra-ready now (still have to review the latest changes). No point in holding up a substantial improvement because there are more details to be improved (there will always be more improvements to be made).